README.md

```diff
@@ -1,14 +1,19 @@
 # Montly Status email
 
-Apps allowing to send monthly status emails to cloud customers. This is a internal apps.
+Send monthly status mails to users. This app doesn't provide a user interface.
+
+Per default it sends a summary of used storage along with some usage hints.
+
+The default messages are in German.
 
 ## Editing the messages sent
 
-All the messages sent to the users are located inside `lib/Service/MessageProvider.php`.
-To overwrite then for internal usage, you need to create a new class inheriting from the
-MessageProvider and then overwrite any methods with your own messages.
+The messages sent to users are defined at `lib/Service/MessageProvider.php`.
+
+To overwrite the default messages, create a new class inheriting from `MessageProvider`
```

Comment:
This step is very strange to me because is a suggestion to do a change inside the app code. But now is the way that this app works. Is a point to think more about and suggest a best implementation.

Reply:
Haha, I fully agree. Actually it proposes to create a new app that provides this class. We should think about improving this in the future by making it configurable in the app. It's just how it's working currently :wink: